spec/requests/providers_spec.rb

```diff
@@ -1837,6 +1837,52 @@ class DummyProvider; end
     end
   end
 
+  context 'configured_systems subcollection' do
+    let(:configured_system) { FactoryBot.create(:configured_system, :manager => ems) }
+    let(:ems) { FactoryBot.create(:configuration_manager) }
+
+    context 'GET /api/providers/:id/configured_systems' do
+      it 'returns the configured_systems with an appropriate role' do
+        api_basic_authorize subcollection_action_identifier(:providers, :configured_systems, :read, :get)
+
+        expected = {
+          'resources' => [{'href' => api_provider_configured_system_url(nil, ems, configured_system)}]
+        }
+        get(api_provider_configured_systems_url(nil, ems))
+
+        expect(response).to have_http_status(:ok)
+        expect(response.parsed_body).to include(expected)
+      end
+
+      it 'does not return the configured_systems without an appropriate role' do
+        api_basic_authorize
+
+        get(api_provider_configured_systems_url(nil, ems))
+
+        expect(response).to have_http_status(:forbidden)
+      end
+    end
+
+    context 'GET /api/providers/:id/configured_systems/:s_id' do
+      it 'returns the configured_systems with an appropriate role' do
+        api_basic_authorize action_identifier(:providers, :read, :resource_actions, :get)
```

Comment:
Lookin' at configuration profiles, I `think` this should be action_identifier(:configured_systems, :read, :subresource_actions, :get)